### DISCUSSION TITLE
Visibility of contest exchange/contest name in QSO view

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#792] [Ideas / Feature Requests]
Currently, I don't think there's a way to see either the contest name or the contest exchange when viewing QSOs in Cloudlog. There's also no way to change a contest name by editing a QSO.

**Comment by magicbug** [comment]
Contest changes have always been shown in the QSO popup next to RST, However Contest ID is new attention in the same window. You are correct in that its not in the edit window
